db/migrate/20190617142328_create_katello_ansible_collections.rb

```diff
@@ -0,0 +1,26 @@
+class CreateKatelloAnsibleCollections < ActiveRecord::Migration[5.2]
+  def change
+    create_table :katello_ansible_collections do |t|
+      t.string :pulp_id, :null => false, :limit => 255
+      t.string :checksum
+      t.string :name
+      t.string :namespace
+      t.string :version
+      t.timestamps
+    end
+
+    add_index :katello_ansible_collections, :pulp_id, :unique => true, :name => 'katello_ansible_collections_pulp_id_index'
+    add_index :katello_ansible_collections, [:id, :pulp_id, :name, :version, :namespace], :name => 'katello_ansible_collections_fields_index'
```

Comment:
Do we really need to index every attribute?

Reply:
Removed indexing for all fields other than pulp_id.